frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.pdf.component.js

```diff
@@ -1,19 +1,611 @@
+        <PositionedString
+          dataKey="person.income.unionDues"
+          top="80.2%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.income.garnishments"
+          top="82.6%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.income.retireDeposits"
+          top="84.8%"
+          left="69.5%"
+        />
+      </RenderPage>
+      <RenderPage url="/static/forms/motion-to-waive-fees/Motion_to_Waive_Fees_and_Statement_Supporting-06.png">
+        <PositionedCheckmark
+          dataKey="person.businessInterest"
+          top="28%"
+          left="18%"
+        />
+        <PositionedCheckmark
+          dataKey="person.noBusinessInterest"
+          top="25.7%"
+          left="18%"
+        />
+        <PositionedCheckmark
+          dataKey="person.financialAssets"
+          top="66.7%"
+          left="18%"
+        />
+        <PositionedCheckmark
+          dataKey="person.noFinancialAssets"
+          top="64.1%"
+          left="18%"
+        />
+      </RenderPage>
+      <RenderPage url="/static/forms/motion-to-waive-fees/Motion_to_Waive_Fees_and_Statement_Supporting-07.png"></RenderPage>
+      <RenderPage url="/static/forms/motion-to-waive-fees/Motion_to_Waive_Fees_and_Statement_Supporting-08.png">
+        <PositionedCheckmark dataKey="person.realEstate" top="29%" left="18%" />
+        <PositionedCheckmark
+          dataKey="person.noRealEstate"
+          top="26.4%"
+          left="18%"
+        />
+        <PositionedCheckmark
+          dataKey="person.personalProperty"
+          top="80.4%"
+          left="18%"
+        />
+        <PositionedCheckmark
+          dataKey="person.noPersonalProperty"
+          top="77.8%"
+          left="18%"
+        />
+      </RenderPage>
+      <RenderPage url="/static/forms/motion-to-waive-fees/Motion_to_Waive_Fees_and_Statement_Supporting-09.png">
+        <PositionedCheckmark
+          dataKey="person.debtsOwed"
+          top="42.8%"
+          left="18%"
+        />
+        <PositionedCheckmark
+          dataKey="person.noDebtsOwed"
+          top="40.3%"
+          left="18%"
+        />
+        <PositionedCheckmark
+          dataKey="person.otherReasonNotToPay"
+          top="75.2%"
+          left="18%"
+        />
+        <PositionedString
+          dataKey="person.otherNoPayExplanantion"
+          top="77.6%"
+          left="18%"
+        />
+      </RenderPage>
+      <RenderPage url="/static/forms/motion-to-waive-fees/Motion_to_Waive_Fees_and_Statement_Supporting-10.png">
+        <PositionedString debugKey="todaysDate" left="11.5%" top="30.6%">
+          {moment().format("L")}
+        </PositionedString>
+        <PositionedString debugKey="printedName" left="54.4%" top="33.6%">
+          {`${renderData("person.firstName")} ${renderData(
+            "person.middleName"
+          )} ${renderData("person.lastName")}`}
+        </PositionedString>
+      </RenderPage>
```

Comment:
this form is such a beast. normally i'd probably recommend we break this up into multiple components (maybe one per page) but In the interest of completing things I think this is 👍

Reply:
Yeah this is a monster form. I think I may break this into smaller components.